app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+
+  def create
+    @company = Company.find(params[:company_id])
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
```

Comment:
This is not the ideal way to create objects that are associated to each other. If we instead find the job first with `@job = Job.find(params[:job_id])` and because a job has many comments, we can do...

```ruby
@comment = @job.comments.new(comment_params)
```